src/ai/backend/testutils/bootstrap.py

```diff
@@ -61,19 +60,19 @@ def etcd_container() -> Iterator[tuple[str, HostPortPair]]:
 def redis_container() -> Iterator[tuple[str, HostPortPair]]:
     # Spawn a single-node etcd container for a testing session.
     redis_allocated_port = 36379 + get_parallel_slot() * 10
+    print('spawning redis container on port', redis_allocated_port)
     proc = subprocess.run(
         [
             'docker', 'run', '-d',
             '-p', f'0.0.0.0:{redis_allocated_port}:6379',
             '--health-cmd', 'redis-cli ping',
             '--health-interval', '1s',
             '--health-start-period', '0.3s',
-            'redis:6.2-alpine',
+            'redis:7-alpine',
```

Comment:
I think we should add a follow-up PR to make the halfstack docker-compose configurations (let's add a `2206` variant and set it as the current) to use Redis 7.0.  Just keep this line for this PR.

Reply:
:+1: